Sateltiles/init.py

```diff
@@ -0,0 +1,11 @@
+import wx
+import login as start
+import admin_user as admin
+import normal_user as norm
+
+if __name__ == '__main__':
+    app = wx.App()
+

```

Comment:
why would you want to use this as stand alone runable script..are you using this anywhere?